demo/src/style/app.scss

```diff
@@ -5,8 +5,9 @@ header.navbar {
 }
 
 .masthead {
-  padding-top: 5rem;
-  padding-bottom: 5rem;
+  a {
+    flex: 1;
```

Comment:
you should probably add `-ms` prefixes, as there is no autoprefixer currently

Reply:
fixed by removing it entirely. It was messing with button sizes on mobile display under IE